core/src/main/scala/scalafix/rewrite/Xor2Either.scala

```diff
@@ -13,16 +14,90 @@ case object Xor2Either extends Rewrite {
     }
     object DType extends Desugared[Type]
     object DTerm extends Desugared[Term]
+
+    //If we start making many of these, would it be worth having case classes with some structure?
+    //Possibly if I could select the appropriate based on a match, I have not been able to do that yet.
+    case class ReplaceType(original: Type,
+                           replacement: Type,
+                           newString: String)
+    val xorT: ReplaceType =
+      ReplaceType(t"cats.data.XorT", t"scala.util.EitherT", "EitherT")
+    val xor: ReplaceType =
+      ReplaceType(t"cats.data.Xor", t"scala.util.Either", "Either")
+    val xorLeft: ReplaceType =
+      ReplaceType(t"cats.data.Xor.Left", t"scala.util.Left", "Left")
+    val xorRight: ReplaceType =
+      ReplaceType(t"cats.data.Xor.Right", t"scala.util.Either.Right", "Right")
```

Comment:
Down the road, we may be able to generalize rewrites so they can be implement from a `Seq[(Tree, Tree)]`. Very exciting.